.github/workflows/swagger.yml

```diff
@@ -0,0 +1,31 @@
+name: Open API Specification and Documentation
+on: [push, pull_request]
+jobs:
+  swaggerci:
+    name: swagger-build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup go-swagger
+        uses: minchao/setup-go-swagger@v1
+        with:
+          version: v0.26.1
+
+  build:
+    needs: [swaggerci]
+    name: Swagger-ci
+    runs-on: ubuntu-latest
+    steps:
+    - name: swagger-build
+      uses: actions/checkout@v2
+      run: |
+          sudo apt install snapd
+          sudo snap install go-swagger
+          sudo apt update
+    - name: swagger-spec
+      run: swagger generate spec -o ./helpers/swagger.yaml --scan-models
+      uses: actions/cache@v1
+      with:

```

Comment:
There is still a lint error:

![image](https://user-images.githubusercontent.com/2797052/124986209-fdf13f80-e000-11eb-8aa3-fa1dffea6946.png)

Remove:
- `uses: actions/cache@v1` and
- `with:`

they are not needed.

Reply:
@ramrodo : Oh Yes, I forgot we doesn't need to store `cache`!
Thanks for reminder, it's done! :slightly_smiling_face: